e2e-tests/tests/monitoring/01-deploy-pmm-server.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |-
+      set -o errexit
+      set -o xtrace
+
+      source ../../functions
+
+      deploy_pmm_server
+
+      api_key=$(generate_pmm_api_key)
+      api_key_encoded=$(echo -n ${api_key} | base64 -w 0)
```

Comment:
Why do we need to use `-w 0`.  My  base64  does not support it at all :(

Reply:
it's different with linux and macos. we have it in our docs too: https://docs.percona.com/percona-operator-for-mysql/pxc/users.html#yaml-object-format

Reply:
https://github.com/percona/percona-postgresql-operator/pull/313/commits/422611dd74c87af724c7bf38f628e6c5286f291a